shanoir-ng-datasets/src/main/java/org/shanoir/ng/configuration/amqp/RabbitMQConfiguration.java

```diff
@@ -48,8 +61,7 @@ public FanoutExchange fanout() {
 	@RabbitListener(bindings = @QueueBinding(

```

Comment:
And this can be set in RabbitMQDatasetService

Reply:
I don't have this class locally.. I should merge with the current code I think

Reply:
Yes and I don't think it's gonna be painless. Don't hesitate if you have a doubt :)